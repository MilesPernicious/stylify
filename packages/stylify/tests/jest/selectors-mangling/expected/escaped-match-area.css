.b{
	color: blue
}
.a{
	font-size: 24px
}
