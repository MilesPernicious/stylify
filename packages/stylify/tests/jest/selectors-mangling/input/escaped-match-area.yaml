# stylify-components title: 'font-size:24px' /stylify-components
lol: "Some <strong class=\"color:blue title\"></strong>"
