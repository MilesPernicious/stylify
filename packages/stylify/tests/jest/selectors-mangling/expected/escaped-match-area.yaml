# stylify-components title: 'font-size:24px' /stylify-components
lol: "Some <strong class=\"b a\"></strong>"
